modules/serverbidBidAdapter.js

```diff
@@ -44,20 +53,29 @@ const ServerBidAdapter = function ServerBidAdapter() {
     '800x250'
   ];
 
+  sizeMap[77] = '970x90';
+  sizeMap[123] = '970x250';
+  sizeMap[43] = '300x600';
+
   const bidIds = [];
 
   baseAdapter.callBids = function(params) {
-    if (params && params.bids && utils.isArray(params.bids) && params.bids.length) {
+    if (params && params.bids &&
+        utils.isArray(params.bids) &&
+        params.bids.length &&
+        CONFIG[params.bidderCode]) {
+      const config = CONFIG[params.bidderCode];
+      config.request = window[params.bidderCode.toUpperCase() + '_CONFIG'];
```

Comment:
@jgrimes What is the purpose of adding this line. This will be set to undefined, i guess your library will set window[params.bidderCode.toUpperCase() + '_CONFIG'];

Reply:
@jaiminpanchal27 It is to allow people using our adapter (or aliases) to add arbitrary additional configuration/information on the requests being made to us.